jvm-packages/scalastyle-config.xml

```diff
@@ -245,33 +268,81 @@ This file is divided into 3 sections:
 
   <!-- Doesn't seem super big deal here ... -->
   <check level="error" class="org.scalastyle.file.FileLengthChecker" enabled="false">
-    <parameters><parameter name="maxFileLength">800></parameter></parameters>
+    <parameters>
+      <parameter name="maxFileLength">800></parameter>
+    </parameters>
   </check>
 
   <!-- Doesn't seem super big deal here ... -->
   <check level="error" class="org.scalastyle.scalariform.NumberOfTypesChecker" enabled="false">
-    <parameters><parameter name="maxTypes">30</parameter></parameters>
+    <parameters>
+      <parameter name="maxTypes">30</parameter>
+    </parameters>
   </check>
 
   <!-- Doesn't seem super big deal here ... -->
   <check level="error" class="org.scalastyle.scalariform.CyclomaticComplexityChecker" enabled="false">
-    <parameters><parameter name="maximum">10</parameter></parameters>
+    <parameters>
+      <parameter name="maximum">10</parameter>
+    </parameters>
   </check>
 
   <!-- Doesn't seem super big deal here ... -->
   <check level="error" class="org.scalastyle.scalariform.MethodLengthChecker" enabled="false">
-    <parameters><parameter name="maxLength">50</parameter></parameters>
+    <parameters>
+      <parameter name="maxLength">50</parameter>
+    </parameters>
   </check>
 
   <!-- Not exactly feasible to enforce this right now. -->
   <!-- It is also infrequent that somebody introduces a new class with a lot of methods. -->
   <check level="error" class="org.scalastyle.scalariform.NumberOfMethodsInTypeChecker" enabled="false">
-    <parameters><parameter name="maxMethods"><![CDATA[30]]></parameter></parameters>
+    <parameters>
+      <parameter name="maxMethods"><![CDATA[30]]></parameter>
+    </parameters>
   </check>
 
   <!-- Doesn't seem super big deal here, and we have a lot of magic numbers ... -->
   <check level="error" class="org.scalastyle.scalariform.MagicNumberChecker" enabled="false">
-    <parameters><parameter name="ignore">-1,0,1,2,3</parameter></parameters>
+    <parameters>
+      <parameter name="ignore">-1,0,1,2,3</parameter>
+    </parameters>
+  </check>
+
+  <check level="error" class="org.scalastyle.scalariform.IllegalImportsChecker" enabled="true">
+    <parameters>
+      <parameter name="illegalImports"><![CDATA[scala.collection.Seq,scala.collection.IndexedSeq]]></parameter>
+    </parameters>
+    <customMessage><![CDATA[
+        Don't import scala.collection.Seq and scala.collection.IndexedSeq as it may bring some problems with cross-build between Scala 2.12 and 2.13.
```

Comment:
```suggestion
        Don't import scala.collection.Seq and scala.collection.IndexedSeq as it may cause issues with cross-build between Scala 2.12 and 2.13.
```